'

```diff
@@ -0,0 +1,89 @@
+# CloudTrail
+
+resource "aws_cloudtrail" "data_refinery_s3_cloudtrail" {
+  name                          = "data-refinery-s3-cloudtrail-${var.user}-${var.stage}"
```

Comment:
Oooh! I know the terraform docs write it with the spacing like this but the convention for our project is to not put more than one space before the `=`. The reasoning is that if you added a longer line to this block then you have to change every other line in the block.